Update project leadership with new PSC members.

iosefa opened the pull request:
Redistributed contributor roles for clarity and consistency. Updated leadership information for pgRouting-related projects and added new contributors with their respective roles.

Changes proposed in this pull request:
- updates PSC on website

@pgRouting/admins


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
	- Updated Project Steering Committee membership
	- Added two new members: Iosefa Percival and Ko Nagase
	- Removed Cayetano Benavent from the committee
	- Adjusted leadership role for vrpRouting project from Leader to Co-leader
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

iosefa committed:
Update project leadership with new PSC members.

Redistributed contributor roles for clarity and consistency. Updated leadership information for pgRouting-related projects and added new contributors with their respective roles.

sanak commented on development.rst:
```diff
@@ -36,6 +28,18 @@ Current members of the the Project Steering Committee are:
+
+  * OSGeo Charter member since 2015
+  * Co-leader of pgRouting Docker
+  * Co-leader of osm2pgRouting
```
Changing this line as follow is helpful for me. 🙇
```diff
-   * Co-leader of osm2pgRouting
+   * Co-leader of pgRoutingLayer
```

iosefa replied:
@sanak thanks! Ive just made the update and sent the request for review again.

sanak reviewed (changes requested):
@iosefa Thanks a lot for update!
I just added comment to my section.

iosefa committed:
Update contribution title from osm2pgRouting to pgRoutingLayer

Corrected the contribution title in the documentation to accurately reflect the role as "Co-leader of pgRoutingLayer" instead of "Co-leader of osm2pgRouting". This ensures consistency and clarity in contributor roles.

sanak reviewed (approved):
Thanks, LGTM!
